spec/unit/application_spec.rb

```diff
@@ -54,6 +54,24 @@
     expect(application.favicon).to eq "/a/favicon.ico"
   end
 
+  it "should store meta tags" do
+    expect(application.meta_tags).to eq({})
+  end
+
+  it "should set meta tags" do
+    application.meta_tags = {author: "My Company"}
+    expect(application.meta_tags).to eq({author: "My Company"})
+  end
+
+  it "should store meta tags for logged out pages" do
+    expect(application.meta_tags_for_logged_out_pages).to eq({})
+  end
+
+  it "should set meta tags for logged out pages" do
+    application.meta_tags_for_logged_out_pages = {robots: "noindex, nofollow"}
+    expect(application.meta_tags_for_logged_out_pages).to eq({robots: "noindex, nofollow"})
```

Comment:
Line is too long. [91/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.